infra/package.json

```diff
@@ -11,6 +11,9 @@
   },
   "license": "MIT",
   "devDependencies": {
+    "@babel/core": "7.23.9",
```

Comment:
might be an issue when incorporating infra to NX

Reply:
Let's hope it'll be a breeze :wind_face: